airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_group_analytics.json

```diff
@@ -0,0 +1,273 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "DATE": {
+      "type": ["null", "string"]
```

Comment:
is it possible to format these date strings as `format: date-time` or is that not the given format from the API? same question for all date fields

Reply:
Added to all date field `format: date` to all schema where it possible.